src/domain/bridge/entities/timing-strategies.ts

```diff
@@ -0,0 +1,2 @@
+// minWaitTime-${minWaitTimeMs}-${startingExpectedResults}-${reduceEveryMs}
+export type TimingStrategies = `minWaitTime-${number}-${number}-${number}`;
```

Comment:
We may want to consider accepting multiple parameters in DTOs to create this, although I don't think it will be required. As with the other remarks, we may remove this.